### DISCUSSION TITLE
安卓14下似乎性能很差

### DISCUSSION THREAD

**Discussion by star-o0O** [eiyooooo/Easycontrol_For_Car#53] [Q&A] [answered]
碰上一件奇怪的事情，我手上有两个设备：
小米平板-安卓13-870-2560x1600
小米14-安卓14-8G3-2670x1200
使用电脑模拟器(安卓9/12)通过wifi控制两台设备，参数为：1920-90fps-12Mbps
小米平板操作流畅，小米14有很大延迟
使用小米14控制小米平板，很流畅
反过来使用小米平板控制小米14，有很大延迟
但是使用wlan直连时，两台设备互相操作都很流畅
这是什么原理？

**Comment by eiyooooo** [comment]
最近在写使用wlan直连的设备控制了，应该国庆左右能发个一版，易控暂时不维护了

**Reply by star-o0O** [reply]
> 最近在写使用wlan直连的设备控制了，应该国庆左右能发个一版，易控暂时不维护了

好的大佬

**Comment by star-o0O** [comment, marked as answer]
看了一下代码，发现为了兼容性调小了参数
`public static final int CONNECT_MAXDATA = 15 * 1024`
我改成原先推荐的
`public static final int CONNECT_MAXDATA = 1024 * 1024`
按上述操作测试了几天，流畅极了
